docs/launch_microbench_test.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+
+#This is a job launch script
+
+import os
+import sys
+from uuid import UUID
+
+from gem5art.artifact.artifact import Artifact
+from gem5art.run import gem5Run
+from gem5art import tasks
+
+"""packer = Artifact.registerArtifact(
```

Comment:
Two problems here :)
1) `'''` is not a good way to comment in python.
2) We should remove all commented code, if it's not needed.